backend/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
-RUN \ 
-apk add --no-cache npm python ffmpeg && \
+ENV UID=991 GID=991
+
+RUN \
+apk add --no-cache npm python2 ffmpeg tini su-exec && \
 apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing/ \
     atomicparsley
 
 WORKDIR /app
 
+COPY docker_run.sh /usr/local/bin/run.sh
```

Comment:
I think it would be less confusing if the file would be named locally run.sh. It is already in a docker directory so there is no confussion.